CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGLOG
 
+## 0.5.3
```

Comment:
This is a breaking change so we should bump the largest non-zero number (`0.6.0`)